src/Libraries/Microsoft.Extensions.AI.Evaluation/ChatMessageExtensions.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI.Evaluation;
+
+/// <summary>
+/// Extension methods for <see cref="ChatMessage"/>.
+/// </summary>
+public static class ChatMessageExtensions
```

Comment:
Do you think these helpers will be stable enough to be made public when we go to GA? Should we mark them as experimental for the first release? or can we make them internal?